api/osb/osb_controller.go

```diff
@@ -136,6 +137,29 @@ func (c *Controller) proxy(r *web.Request, logger *logrus.Entry, broker *types.S
 	return validateBrokerResponse(recorder, broker)
 }
 
+func modifyRequestURLPath(ctx context.Context, m []string, modifiedRequest *http.Request) {
+	if isInstanceSharingFlow(ctx) {
+		modifiedRequest.URL.Path = getPathForInstanceSharing(ctx, m[1])
```

Comment:
1.why do we modify the  OSB Request URL Path in case an instance is shared?
2.what will happen in case an instance is provisioned as a shared instance? (with the shared property=true)

Reply:
As discussed, I committed a change with the "SharedInstanceFromContext" change.

Reply:
Now we do that just for binding.

Reply:
I resolve that case since it was discussed already.